components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/edit.jsp

```diff
@@ -67,6 +68,11 @@
     List<String> allowedScopeValidators = new ArrayList<String>();
     List<String> scopeValidators = new ArrayList<String>();
 
+    List<String> supportedEncryptionAlgorithms =
+            OAuthServerConfiguration.getInstance().getSupportedIdTokenEncryptionAlgorithm();
+    List<String> supportedEncryptionMethods =
+            OAuthServerConfiguration.getInstance().getSupportedIdTokenEncryptionMethods();
+
```

Comment:
Here backend APIs are directly used adding dependencies for the backend component. Shouldn't this be invoked over a service API. Could you please check the common practice being used in other UI bundles

Reply:
I have changed the approach. Please review. :)